{"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["Linux"],"tokens":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"languages":["swift"]}]}],"schemaVersion":{"major":0,"patch":0,"minor":3},"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"additional context information related to the error"}],"variants":[{"paths":["\/documentation\/sotocore\/awsresponseerror\/context"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"kind":"symbol","metadata":{"role":"symbol","externalID":"s:8SotoCore16AWSResponseErrorV7contextAA15AWSErrorContextVSgvp","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"modules":[{"name":"SotoCore"}],"roleHeading":"Instance Property","title":"context","symbolKind":"property"},"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"navigatorTitle":[{"text":"AWSErrorContext","kind":"identifier"}],"url":"\/documentation\/sotocore\/awserrorcontext","type":"topic","kind":"symbol","title":"AWSErrorContext","role":"symbol","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AWSErrorContext","kind":"identifier"}],"abstract":[{"text":"Additional information about error","type":"text"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext"},"doc://codes.soto.soto-core/documentation/SotoCore":{"abstract":[{"type":"text","text":"The core framework for Soto, a Swift SDK for AWS"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","title":"SotoCore","role":"collection","type":"topic","url":"\/documentation\/sotocore","kind":"symbol"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError/context":{"kind":"symbol","title":"context","abstract":[{"type":"text","text":"additional context information related to the error"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context","role":"symbol","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"text":"AWSErrorContext","kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"text":"?","kind":"text"}],"type":"topic","url":"\/documentation\/sotocore\/awsresponseerror\/context"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError":{"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"AWSResponseError"}],"abstract":[{"type":"text","text":"Response Error type returned by Soto if the error code is unrecognised"}],"kind":"symbol","type":"topic","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError","url":"\/documentation\/sotocore\/awsresponseerror","title":"AWSResponseError","navigatorTitle":[{"kind":"identifier","text":"AWSResponseError"}],"role":"symbol"}}}